usr/bin/revpi-btuart

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth attach for Revolution Pi BT
+. /usr/bin/revpi-common
+
+HCIATTACH=/usr/bin/hciattach
+TTYDEVNAME="$1"
+
+revpi_dev_type
+DTYPE=$?
+
+if [ "0" = $DTYPE ]; then
+  OPTS=""
+  TYPEID="any"
+  SPEED=""
+  FLOW=""
+  BDADDR=""
```

Comment:
These don't make sense. Adding a BDADDR without the keywork bdaddr doesn't work.